### ISSUE TITLE
Add a warning when no links are found instead of loading the player

### ISSUE THREAD

**Issue by meenbeese** [recloudstream/cloudstream#340]
### Describe your suggested feature

Currently, when there are no links for a film, there is no way to know this is the case. When you tap “play”, the app loads the film for a few minutes as if the film is about to start. Only after this is there a toast message that reads “no link found” etc. There is no easy way to know if the film exists and is just lagging or it does not exist. This is an infuriating issue. It should be fairly easy for the app to display this message before waiting for 2-3 minutes. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by LagradOst** [comment]
cant be solved
